### PR TITLE
Update config and binaries for latest changes in OpenTitan

### PR DESCRIPTION
**Pull request by vogelpi**
The main clock of the target is now 10 MHz and we use that directly for the ADC on the capture board.

### CODE REVIEW

**Comment by vogelpi** [comment]
The binaries included in this PR are much more current but some of the changes are not yet in OpenTitan master.

Regarding the `spiflash` binary: I can't run this as I have the wrong glibc installed here (Ubuntu GLIBC 2.27-3ubuntu1.2 vs. GLIBCXX_3.4.26). Can we include a statically linked version of `spiflash`?

File: cw/cw305/capture.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 device:
   fpga_bitstream: objs/lowrisc_systems_top_earlgrey_cw305_0.1.bit
   fw_bin: objs/aes_serial_fpga_nexysvideo.bin
-  pll_frequency: 25000000
-  baudrate: 57600
+  pll_frequency: 100000000
```
Comment (moidx) [inline cw/cw305/capture.yaml:4]: nice!

**Review by moidx — APPROVED** [review]
LGTM. 

I will look at statically compiling spiflash.

**Review by alphan — APPROVED** [review]

**Comment by vogelpi** [comment]
> I will look at statically compiling spiflash.

That would be awesome!

Thanks guys for the approval.

**Comment by moidx** [comment]
Lets check this in as is. 

I think we need to discuss in more details the implications of maintaining a static library, and alternative options.

**Pull request merged**